plugins/inlinetoolbar/skins/moono/inlinetoolbar.css

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+.cke_balloon.cke_inlinetoolbar
+{
+	background: linear-gradient(to bottom, #f5f5f5, #cfd1cf);
+	border: 1px solid #b6b6b6;
+	-webkit-border-radius: 0px;
+	-moz-border-radius: 0px;
+	border-radius: 0px;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_content
+{
+	min-height: 10px;
+}
+
+/* side: [ bottom, top ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_bottom,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_top
+{
+	border-color: #b6b6b6 transparent;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_top,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_bottom
+{
+	border-color: #cfd1cf transparent;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_bottom,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_top
```

Comment:
Why isn't it merged with previous item as it affects identical elements (has the same selectors)?